tests/util/simple_tests.rs

```diff
@@ -25,22 +25,50 @@ use self::tempdir::TempDir;
 use self::uuid::Uuid;
 
 use libstratis::engine::{Engine, EngineError, ErrorEnum};
+use libstratis::engine::types::{FilesystemUuid, Redundancy};
 use libstratis::engine::strat_engine::blockdevmgr::{initialize, resolve_devices};
 use libstratis::engine::strat_engine::device::{blkdev_size, wipe_sectors, write_sectors};
 use libstratis::engine::strat_engine::engine::DevOwnership;
 use libstratis::engine::strat_engine::filesystem::{create_fs, mount_fs, unmount_fs};
 use libstratis::engine::strat_engine::metadata::{StaticHeader, BDA_STATIC_HDR_SECTORS,
                                                  MIN_MDA_SECTORS};
-use libstratis::engine::strat_engine::pool::{get_dmdevs, get_filesystems};
+use libstratis::engine::strat_engine::pool::{get_dmdevs, get_filesystems, DATA_BLOCK_SIZE,
+                                             INITIAL_DATA_SIZE, StratPool};
 use libstratis::engine::strat_engine::serde_structs::Recordable;
 use libstratis::engine::strat_engine::setup::{find_all, get_blockdevs, get_metadata};
 use libstratis::engine::strat_engine::StratEngine;
+use libstratis::engine::Pool;
 
 /// Dirty sectors where specified, with 1s.
 fn dirty_sectors(path: &Path, offset: Sectors, length: Sectors) {
     write_sectors(path, offset, length, &[1u8; SECTOR_SIZE]).unwrap();
 }
 
+/// Verify that a the physical space allocated to a pool is expanded when
+/// the nuber of sectors written to a thin-dev in the pool exceeds the
+/// INITIAL_DATA_SIZE.  If we are able to write more sectors to the filesytem
+/// than are initially allocated to the pool, the pool must have been expanded.
+fn verify_pool_expand(pool: &mut StratPool, fs_uuid: &FilesystemUuid) {
+    let devnode = pool.get_filesystem(&fs_uuid)
+        .unwrap()
+        .devnode()
+        .unwrap();
+    let mut f = OpenOptions::new().write(true).open(devnode).unwrap();
+    // Write 10 percent more sectors than is initially allocated to a pool
+    let write_size: u64 = (INITIAL_DATA_SIZE.0 as f64 +
```

Comment:
Why not just make ```write_size``` double the initial? If the amount is just doubled, that should test expansion once. If the algorithm is refined, so that the increase is less than double, then the test will just test increase over multiple iterations, which ought to be good.